src/frontend/src/lib/canisters/signer.errors.ts

```diff
@@ -33,3 +34,18 @@ export const mapSignerCanisterBtcError = (err: SignerCanisterBtcError): Canister
 	}
 	return new CanisterInternalError('Unknown SignerCanisterBtcError');
 };
+
+export const mapSignerCanisterGetEthAddressError = (
+	err: GenericSigningError
+): CanisterInternalError => {
+	if ('SigningError' in err) {
+		const [code, addOns] = err.SigningError;
+		return new CanisterInternalError(`Signing error: ${JSON.stringify(code)} ${addOns}`);
```

Comment:
So far we have not localized those errors. The PR follows the existing pattern. Given that they are thrown at a low level, we can assume that if the error should be beautified this can be done at the service layer? Maybe something we can discuss.

Reply:
cool! then this is ok